### DISCUSSION TITLE
能否维护下 ghcr.io/halo-dev/halo 镜像的 latest 标签

### DISCUSSION THREAD

**Discussion by sunist-c** [halo-dev/halo#7179] [Q&A] [answered]
需要将 halo 镜像同步到内网 registry 中，自动脚本定时拉 latest，发现拉了很久一直是 1.6.1 版本......

**Comment by ruibaby** [comment, marked as answer]
Halo 2 不提供 latest 标签是防止 Halo 1.x 用户误更新，如果你需要自动同步 Halo 2 的所有标签，可以用 ghcr.io/halo-dev/halo:2
